From 0b62c7a5c4128ca52b5de079c1f3da9c08ba48ba Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Wed, 28 Feb 2018 21:25:22 -0800
Subject: [PATCH 05/23] test-sizeof.c: Disable tests for missing typedefs in
 musl

Upstream-Status: Inappropriate [musl specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/test/test-sizeof.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/test/test-sizeof.c b/src/test/test-sizeof.c
index 55bd81e22f..6c6ae5bd01 100644
--- a/src/test/test-sizeof.c
+++ b/src/test/test-sizeof.c
@@ -55,8 +55,11 @@ int main(void) {
         info(unsigned);
         info(unsigned long);
         info(unsigned long long);
+
+#ifdef __GLIBC__
         info(__syscall_ulong_t);
         info(__syscall_slong_t);
+#endif
         info(intmax_t);
         info(uintmax_t);
 
@@ -76,13 +79,17 @@ int main(void) {
         info(ssize_t);
         info(time_t);
         info(usec_t);
+#ifdef __GLIBC__
         info(__time_t);
+#endif
         info(pid_t);
         info(uid_t);
         info(gid_t);
         info(socklen_t);
 
+#ifdef __GLIBC__
         info(__cpu_mask);
+#endif
 
         info(enum Enum);
         info(enum BigEnum);
-- 
2.39.0

