From b4c7bc904069e5ef0e8baae0fc85b40182143684 Mon Sep 17 00:00:00 2001
From: Tilmann Meyer <me@atiltedtree.dev>
Date: Thu, 6 Jan 2022 23:09:06 +0100
Subject: [PATCH 15/19] add missing include

---
 src/shared/mkdir-label.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/shared/mkdir-label.c b/src/shared/mkdir-label.c
index d36a6466d7..5b1ac5d1e0 100644
--- a/src/shared/mkdir-label.c
+++ b/src/shared/mkdir-label.c
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <sys/stat.h>
+
 #include "mkdir-label.h"
 #include "selinux-util.h"
 #include "smack-util.h"
-- 
2.35.1

